src/System.Xml.Serialization/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+DEFAULT_APPNAME Error: 0 : Unable to resolve assembly 'Assembly(Name=mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)' referenced by the contract assembly 'Assembly(Name=System.Xml.Serialization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)'.
```

Comment:
I believe this was happening because apicompat wasn't finding mscorlib in the same directory as the assembly it was checking compat for(in this case System.Xml.Serialization). Since I'm changing the facades to be code-driven and use a project-to-project reference, now the output dir will have not only the built assembly but all of its dependencies as well. I have validated this by starting to work on some facades, and the error did go away.